lib/src/sv_onvif.h

```diff
@@ -114,4 +116,13 @@ void
 onvif_media_signing_authenticity_report_free(
     onvif_media_signing_authenticity_t *authenticity_report);
 
+// Common to Siging and Validation
```

Comment:
Can you put these common APIs before the signing and validation specific APIs, please.

nit: Siging -> Signing